src/config/rcxml.c

```diff
@@ -1076,7 +1111,8 @@ entry(xmlNode *node, char *nodename, char *content)
 		return true;
 
 	} else if (str_space_only(content)) {
-		/* ignore empty leaf nodes other than above */
+		wlr_log(WLR_ERROR, "Empty string is currently not "
```

Comment:
"Empty string is current not allowed" doesn't feel quite right because:

- I would argue that they are not "not allowed", we just interpret them as nothing by ignoring them.
- And "currently" feels like we're in a temporary state, which I prefer not to be on this point. In other words, I feel we ought to make up our mind which way we're going and then stick with it.